language/tools/move-cli/tests/testsuite/explain_arithmetic_failure/args.exp

```diff
@@ -1,2 +1,10 @@
 Command `run script.move`:
+<<<<<<< HEAD
```

Comment:
this cannot be right is it? 😄

Reply:
The CLI tests were broken but have now been fixed, I need to rebase :).